devops/ansible/playbook.yml

```diff
@@ -233,6 +233,9 @@
       DISPLAY: :7
     when: arbor
 
+  - name: arbor | install logo
+    get_url: url=https://arbor.kitware.com/arbor.jpg dest=/vagrant/app/arbor.jpg
```

Comment:
On second thought it may be good to add the Arbor and Tangelo logos to this repo and just pull it from the filesystem - seems much safer.